quirks/password-rules.json

```diff
@@ -216,6 +216,9 @@
     "macys.com": {
         "password-rules": "minlength: 7; maxlength: 16; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789~!@#$%^&*+`(){}[:;\"'<>?]];"
     },
+    "makemytrip.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [@$!%*#?&];"
```

Comment:
Can you provide some primary source information about how you derived this rule? It looks like a good rule, and I’m happy to merge, but we’d like to have this information on hand.

Reply:
![](https://i.ibb.co/jDGHw1M/2020-07-04-22-11.png)